Source/Core/DiscIO/WbfsBlob.h

```diff
@@ -19,7 +19,10 @@ class WbfsFileReader : public IBlobReader
 public:
 	static WbfsFileReader* Create(const std::string& filename);
 
-	u64 GetDataSize() const override { return m_size; }
+	// The WBFS format does not save the original file size.
+	// This function returns a constant upper bound
+	// (the size of a double-layer Wii disc).
```

Comment:
Yes, it could be useful for that.

Being stuck with a guessing method that's wrong would be worse than being stuck with the maximum size.